src/tag/style.js

```diff
@@ -0,0 +1,59 @@
+import styled, { css } from 'styled-components';
+import Close from '../../assets/close.svg';
+
+const DarkBg = css`
+  background-color: #f0f0f8;
+`;
+
+const HideBorder = css`
+  border: none;
+`;
+
+export const Container = styled.div`
+  text-decoration: none !important;
```

Comment:
Is the `!important` necessary -- What is causing it to ignore the css? 
Maybe something is overriding it here that has more specific rule?